### ISSUE TITLE
Warning in logs

### ISSUE THREAD

**Issue by weado** [FreeTHX/remora-ha#4]
Hello, thank to you, because since the 1 year, I use your component to manage my heating and my power consumption.

Since a few months, I have a lot of warning and I would like to share with you, if you know why. For every scan (30 second by default), I have this error message and I tried some fix, but it's the same. 
`XXXX-XX-XX XX:XX:XX WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for remora doing I/O at custom_components/remora/__init__.py, line 84: return self._remora.getTeleInfo()
`
Thank you for your help

**Comment by weado** [comment]
For completness, I have 1 message per get* (GetRelais, GetTeleinfo and GetAllFilPilote)
`WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for remora doing I/O at custom_components/remora/__init__.py, line 125: return self._remora.getRelais()
WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for remora doing I/O at custom_components/remora/__init__.py, line 84: return self._remora.getTeleInfo()
WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for remora doing I/O at custom_components/remora/__init__.py, line 103: return self._remora.getAllFilPilote()
`
Is the same for you ? I try on HA 0.110 until 0.112 and it's the same

**Comment by FreeTHX** [comment]
Hi weado,

Thx for your report.
I'm ,sadly, aware of this issue.
I will work on it as soon as possible and try to fix it before fall 2020.
Main work will be : a "true async" librairy for remora.

**Comment by FreeTHX** [comment]
Hi @weado 

I finaly took time to fix it.
As recommended in Home Assistant documentation, I/Os have been moved to async_update/update methods.
More, I made it fully async and also updated the python library (pyremona) with async only methods (using aiohttp).

It seems to work fine (1 month of testing)
So, the updated version has been published.
Thx for your feedback.

Next step will be the new configuration system (no more configuration.yaml)
